docs/source/home.rst

```diff
@@ -0,0 +1,63 @@
+:html_theme.sidebar_secondary.remove:
```

Comment:
It will have to be built into the parent folder (not inside Python), and images would have to come from the subfolder (python/_images/...)